platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/InputValidation.java

```diff
@@ -97,4 +105,22 @@ public static boolean checkForClientSideVulnerableMimeType(String mimetype) {
         }
         return true;
     }
+
+    /**
+     * Checks for filenames that have been disallowed by the system.
+     *
+     * @param filename
+     * @return true if the filename is acceptable
+     */
+    public static boolean isFileNameClientSideVulnerable(String filename) {
```

Comment:
❗️ The same mismatch between method name and method semantics is present in this new method. Either switch the name or switch the semantics.